docs/README.md

```diff
@@ -5,7 +5,7 @@ CLI tools
 |Tool name|Description|
 | :--- | :--- |
 |[`android-app-bundle`](android-app-bundle/README.md)|Manage Android App Bundles using     [Bundletool](https://developer.android.com/studio/command-line/bundletool)|
-|[`app-store-connect`](app-store-connect/README.md)|Utility to download code signing certificates and provisioning profiles     from Apple Developer Portal using App Store Connect API to perform iOS code signing|
+|[`app-store-connect`](app-store-connect/README.md)|Interact with Apple services via App Store Connect API|
```

Comment:
Why "Apple services" over "Apple Developer Portal"? I thought the latter is the overarching term for everything related to Apple development.

Reply:
Apple Developer Portal is only connected with code signing... it's not the overarching term for all Apple development functionality.